docs/GettingStartedDocs/visualstudio_dev.md

```diff
@@ -0,0 +1,24 @@
+Open Enclave Wizard - Preview
+=============
+
+This Visual Studio extension includes preview support for Trusted Execution Environment (TEE) platforms,
+including ARM TrustZone and Intel SGX, with a Windows or Linux host application. In addition, this preview
+includes support for testing your enclave under simulation when developing for SGX or TrustZone.
+
+The following matrix shows what combinations are supported in this version:
+```
+OS       Platform           Mode         Supported?
+-------  ----------------   -----------  ----------
+Windows  Intel non-SGX      Simulation   <needs testing>
```

Comment:
Currently `oehost` library depends on Intel SGX dlls. Therefore enclaves cannot be loaded in non-sgx machines, even in simulation mode.

Reply:
Issue #2717 tracks that.  I can change <needs testing> to "Not yet".